packages/desktop/views/account/multi-step/ShowOneTimeAddress.svelte

```diff
@@ -1 +1,69 @@
-<h1>ShowOneTimeAddress</h1>
+<script lang="ts">
+    import { Button, FontWeight, Text, TextType } from 'shared/components'
+    import { localize } from '@core/i18n'
+    import { setClipboard } from '@core/utils'
+
+    let isAddressShown: boolean = false
+
+    const DUMMY_ADDRESS = 'iota1q9f0mlq8yxpx2nck8a0slxnzr4ef2ek8f5gqxlzd0wasgp73utryjtzcp98'
```

Comment:
The comment of the `onCopyClick` function should be put here, since the constant is not only used in the function